lib/ansible/modules/system/aix_lvol.py

```diff
@@ -0,0 +1,354 @@
+            if lv_size > this_vg['free']:
+                module.fail_json(msg="Not enough free space in volume group %s: %s MB free." % (this_vg['name'], this_vg['free']))
+
+            ### create LV
+            mklv_cmd = module.get_bin_path("mklv", required=True)
+
+            cmd = "%s %s -t %s -y %s -c %s  -e %s %s %s %sM %s" % (test_opt, mklv_cmd, lv_type, lv, copies, lv_policy, opts, vg, lv_size, pv_list)
+            rc, _, err = module.run_command(cmd)
+            if rc == 0:
+                module.exit_json(changed=True, msg="Logical volume %s created." % lv)
+            else:
+                module.fail_json(msg="Creating logical volume '%s' failed" % lv, rc=rc, err=err)
+    else:
+        if state == 'absent':
+            ### remove LV
+            rmlv_cmd = module.get_bin_path("rmlv", required=True)
+            rc, _, err = module.run_command("%s %s -f %s" % (test_opt, rmlv_cmd, this_lv['name']))
+            if rc == 0:
+                module.exit_json(changed=True, msg="Logical volume %s deleted." % lv)
+            else:
+                module.fail_json(msg="Failed to remove logical volume %s" % (lv), rc=rc, err=err)
+        else:
+            if this_lv['policy'] != policy:
+                ### change lv allocation policy
+                chlv_cmd = module.get_bin_path("chlv", required=True)
+                rc, _, err = module.run_command("%s %s -e %s %s" % (test_opt, chlv_cmd, lv_policy, this_lv['name']))
+                if rc == 0:
+                    module.exit_json(changed=True, msg="Logical volume %s policy changed: %s." % (lv, policy))
+
+            if vg != this_lv['vg']:
+                module.fail_json(msg="Logical volume %s already exist in volume group %s" % (lv, this_lv['vg']))
+
+            # from here the last remaining action is to resize it, if no size parameter is passed we do nothing.
+            if not size:
+                module.exit_json(changed=False, msg="Logical volume %s already exist." % (lv))
+
+            ### resize LV based on absolute values
+            if int(lv_size) > this_lv['size']:
+                extendlv_cmd = module.get_bin_path("extendlv", required=True)
+                cmd = "%s %s %s %sM" % (test_opt, extendlv_cmd, lv, lv_size - this_lv['size'])
+                rc, out, err = module.run_command(cmd)
+                if rc == 0:
+                    module.exit_json(changed=True, msg="Logical volume %s size extended to %sMB." % (lv, lv_size))
+                else:
+                    module.fail_json(msg="Unable to resize %s to %sMB" % (lv, lv_size), rc=rc, err=err)
+            elif lv_size < this_lv['size']:
+                module.fail_json(msg="No shrinking of Logical Volume %s permitted. Current size: %s MB" % (lv, this_lv['size']))
+            else:
+                module.exit_json(changed=False, msg="Logical volume %s size is already %sMB." % (lv, lv_size))
+
+# import module snippets
+from ansible.module_utils.basic import *
```

Comment:
Please put all imports together at the top, and include only the stuff you are using, i.e.
```python
from ansible.module_utils.basic import AnsibleModule
```